cmd/cniplugins/passt-binding/passt-binding-ds.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: passt-binding-cni
+  namespace: kubevirt
```

Comment:
Is there a need to specify the namespace?

Reply:
aligning to practices we widely use on CNAO
and to https://access.redhat.com/articles/7067964

whoever consume it, can kustomize the namespace adapt it to upstream / downstream

here it is used to be deployed as part of kubevirt upstream for testing purposes, so it is better to put it on kubevirt namespace, which is also cleaned as part of cluster-clean

it is not good practice to dump it on default ns, and no need to use -n for the above reasons

Reply:
Thank you for the explanation @oshoval .

> and to https://access.redhat.com/articles/7067964

This link is not accessible to the public.